content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -1086,6 +1086,23 @@ has been truncated to 1000.
 
 If the number of backend endpoints falls below 1000, the control plane removes this annotation.
 
+### control-plane.alpha.kubernetes.io/leader
+
+Type: Annotation
+
+Example: `control-plane.alpha.kubernetes.io/leader={"holderIdentity":"controller-0","leaseDurationSeconds":15,"acquireTime":"2023-01-19T13:12:57Z","renewTime":"2023-01-19T13:13:54Z","leaderTransitions":1}`
```

Comment:
I'm afraid this isn't a valid representation of a YAML snippet. Try copying this from an actual Endpoints manifest.

Reply:
@sftim  this is not yaml snippet rather it is outcome of kubectl get command corresponding to this annotation

Reply:
Thing is: all the other labels and annotations have their examples in the format of a YAML snippet. Let's follow that convention.

Reply:
@sftim  as its got deprecated i am not able to find exact example in any yaml file. 
Can you help me in this?

Reply:
@sftim  as its got deprecated i am not able to find exact example in any yaml file. 
Can you help me in this?

Reply:
You don't need to copy this from an actual Endpoints; another way is to make sure you are writing [a fragment of] valid YAML.

Try this:
https://learnxinyminutes.com/docs/yaml/

You've got the string value right, but you haven't quoted it, and you've used an `=` where YAML uses a `:`.